pinot-spi/src/main/java/org/apache/pinot/spi/data/FieldSpec.java

```diff
@@ -368,6 +368,10 @@ public Object convert(String value) {
           throw new UnsupportedOperationException("Unsupported data type: " + this);
       }
     }
+
+    public boolean isFixedWidth() {
```

Comment:
Let us add a comment here, so that users are not confused if this refers to fixed length bytes like HLL.

Reply:
Done